data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+			`	Following the doctor's instructions, you quickly exit the room. As you close the door, you hear something of a chuckle, probably from James.`
+				goto end
+			label wait
+			`	Soon enough, a doctor passes by, heading for the room.`
+			`	"Good day! How can I help you?", he inquires.`
+			`	"I just wanted to check on a friend of mine, James, who I heard is in this room. May I see him?"`
+			`	"Well, he is infected by some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James's room.`
+			label end
+			choice
+				`	(Wait outside the room.)`
+			`	Around half an hour later, the doctor finally exists the room.`
+			`	"I see you probably want to help," he says, "I need someone to quickly go to the spaceport and find a captain willing to bring some newly-discovered medication from <destination> as quickly as possible. Can you do that?"`
+			choice
+				`	"Actually I am a spaceship captain. I could do that."`
+				`	"Yes, I can find a capable spaceship captain."`
+					goto `second chance`
+				`	"What am I, a message-boy? Find someone yourself."`
+					decline
+			`	"Indeed? Superb! Come back as soon as possible. We wouldn't want to risk your friend's life for a minute delay."`
+				accept
+			label `second chance`
+			`	On the way to the spaceport you realize that you do not have to find another spaceship captain to bring medicine from <planet>. You could do it yourself!`
+			choice
+				`	(Take the job.)`
+					accept
+				`	(Find someone else.)`
+					decline
+	on accept
+		event "james medicine too slow" 20
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 20
+		system "Alpha Hydri"
+		fleet "Small Northern Pirates"
+
+event "james medicine too slow"
+
+mission "James Medicine [1]"
+	landing
+	name "Medicine from <origin>"
+	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
+	cargo "Essential medicine" 1
+	deadline
+	source "Nimbus"
+	destination "Hestia"
+	to offer
+		has "James Medicine [0]: done"
+	blocked "You have reached <origin>, but you need <capacity> in order to take the medicine for James. Return here when you have the required space free."
+	on visit
+		dialog "generic cargo on visit"
+	on offer
+		dialog `You take the medicine from a newly-constructed section of the spaceport, as instructed. Now you just have to deliver it back to <planet>.`
+	on complete
+		conversation
+			`Landing again on <planet>, you hurry to the Pilot Veteran Clinic with the medicine from <origin>.`
+			branch slow
+				has "event: james medicine too slow"
```

Comment:
what's the point of this event?

Reply:
It makes it so that if you complete the mission with a jump drive, you get the "better" ending, but if you complete it only based on the deadline (or otherwise with just a hyperdrive), you get the "bad" ending.